src/redux/slices/AudioPlayer/state.ts

```diff
@@ -71,21 +72,20 @@ export const loadAndPlayAudioFile = createAsyncThunk<void, number>(
  * @param {number} verseKey example 1:1 -> al-fatihah verse 1
  *
  */
-export const playVerse = createAsyncThunk<void, string>(
+export const playVerse = createAsyncThunk<void, Verse>(
```

Comment:
refactor `playVerse` to `playFrom` that takes: (timestamp, chapterNumber, reciter). 

If the chapterNumber or Reciter do not match, fetch the new audio file.

If it does, set current time. 

This will be the function we use to play a specific word as well, so it's not specific to verses.